Redirected the url to the Jfrog sign up page. (#109)
jfrog/jfrog-docker-desktop-extension: README.md

Follow these directions to proceed, based on your use case:

- If you already have a JFrog environment which includes Xray, all you need to do is set the connection details through the UI.

> - If you don't have a JFrog environment, JFrog supports setting up an environment with a free trial. The extension will then connect to this environment automatically. Please note, if no payment method is added, that this new JFrog environment will not be available after the trial period ends.

Reviewer: @asafcjfrog asked to shorten to:
```suggestion
-  If you don't have a JFrog environment, you can try it out with JFrog's free trial.
```
